roles/pulp_rpm_prerequisites/README.md

```diff
@@ -18,9 +18,10 @@ Here's an example playbook for using pulp_rpm_prerequisites as part of pulp_inst
     ---
     - hosts: all
       vars:
-        pulp_default_admin_password: password
+        pulp_default_admin_password: << YOUR PASSWORD HERE >>
         pulp_settings:
-          secret_key: secret
+          secret_key: << YOUR SECRET HERE >>
+          content_origin: "http://{{ ansible_fqdn }}"
         pulp_install_plugins:
```

Comment:
Funny that you tagged me in this PR @mikedep333 today I opened this issue: https://pulp.plan.io/issues/7065 and this `<< YOUR PASSWORD HERE >> ` reminded me of the divergence between the Pulp core Ansible installer entry and these docs. If anyone can answer the questions and greenlight that issue when you have time, I'll happily fix those docs.

Reply:
@melcorr Thank you for the reminder.